packages/core/test/lib/commands/debug.js

```diff
@@ -36,14 +36,14 @@ describe("debug", function () {
     it("should create networks item in config", function () {
       result = mergeConfigNetwork(config, options);
 
-      assert.notEqual(result.networks.inline_config, undefined);
-      assert.equal(result.networks.inline_config.url, "http://urlhost:1234");
-      assert.equal(result.networks.inline_config.network_id, "*");
+      assert.notEqual(result.networks["urlhost:1234"], undefined);
```

Comment:
It's better to use a variable for `"urlhost:1234"` because it's more than twice. One name would be `expectedNetworkName` that's derived from the given `url` in the `beforeEach`

Reply:
Oh! yes, sure. This makes sense too.